common/test/acceptance/tests/lms/test_lms_dashboard.py

```diff
@@ -81,3 +86,154 @@ def test_dashboard_social_sharing_feature(self):
         self.assertEqual(facebook_widget.attrs('target')[0], '_blank')
         self.assertIn(facebook_url, facebook_widget.attrs('href')[0])
         self.assertIn(facebook_url, facebook_widget.attrs('onclick')[0])
+
+
+class LmsDashboardCourseDateTest(UniqueCourseTest):
+    """ Test suite for the LMS Student Dashboard page """
+    def setUp(self):
```

Comment:
why we can not write a test inside LmsDashboardPageTest class. In that we don't have to setup the course.